doc/en/user/source/rest/api/userrole.rst

```diff
@@ -1,7 +1,7 @@
 .. _rest_api_user_roles:
 
-Users/Groups and Roles Rest API
-===============================
+Users/Groups and Roles
+======================
```

Comment:
REST API examples are in rst, the actual endpoint documentation is in swagger.

We are going through and reviewing swagger docs on geoserver-devel prior to the release; if you can commit to taking part in that process I would be happy to merge this in for the code freeze.

Reply:
@jodygarnett sorry, I somehow missed this review. Thanks. Understanding I now have to wait until after the code freeze for merge. So I should move this documentation to swagger, or how can I help? Thanks.

Reply:
Code is not frozen, will discuss in geoserver meeting.

As above likely we can merge if you can get the docs in before the release